app/views/user_sessions/security_notification.html.haml

```diff
@@ -0,0 +1,31 @@
+= content_for :title, 'Search.gov Login'
+= content_for :navigation do
+  %ul.nav.signup
+    %li= link_to 'Sign up', signup_path
+
+#user-session-form
+  %center
+    %h2 Security Notification
+    %h2 Search.gov is a service of GSA
+    %p for managing search experiences on federal government websites.
+    %p
+      %strong Members of the general public
```

Comment:
Remove extra space after "public". This appears on the page as `Members of the general public , please visit ...`.

Reply:
ah it was not an extra space after public. Was a haml syntax.  (one day...  I will change this all to erb)